.github/workflows/update-gatewayrs.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
i don't think repository_dispatch or workflow_dispatch actions have to be in the master branch to work